tdrs-backend/tdpservice/users/models.py

```diff
@@ -16,12 +16,21 @@ class User(AbstractUser):
     stt = models.ForeignKey(STT, on_delete=models.CASCADE, blank=True, null=True)
     region = models.ForeignKey(Region, on_delete=models.CASCADE, blank=True, null=True)
 
-    # The unique `sub` UUID from decoded login.gov payloads.
+    # The unique `sub` UUID from decoded login.gov payloads for login.gov users.
     login_gov_uuid = models.UUIDField(editable=False,
                                       blank=True,
                                       null=True,
                                       unique=True)
 
+    # Unique `hhsid` user claim for AMS OpenID users.
+    # Note: This field, while currently implemented, is *not* one returned by AMS.
+    # In the future, `TokenAuthorizationAMS.get_auth_options` will use `hhs_id` as the primary auth field.
+    # See also: CustomAuthentication.py
+    hhs_id = models.UUIDField(editable=False,
```

Comment:
As mentioned above, this isn't currently being used but it is ready to be in the future.